draft-packages/menu/KaizenDraft/Menu/components/MenuList.tsx

```diff
@@ -9,7 +9,7 @@ type MenuListProps = {
 
 const MenuList = (props: MenuListProps) => {
   const { heading, children } = props
-  const listHeadingID = uuid()
+  const listHeadingID = v4()
```

Comment:
I don't think it matters for the functionality, but shouldn't this rather be stored in `useState`, otherwise the ID changes each re-render.

Reply:
Ah! Yes it should. Good catch.